internal/statemachine/classic_states.go

```diff
@@ -402,7 +402,7 @@ func generateMountPointCmds(mountPoints []*mountPoint, scratchDir string) (allMo
 		allMountCmds = append(allMountCmds, mountCmds...)
 		allUmountCmds = append(umountCmds, allUmountCmds...)
 	}
-	return
+	return allMountCmds, allUmountCmds, err
```

Comment:
ah, here it is. This commit should probably be squashed with the original one.

Reply:
Indeed :).